super_user/migrations/0001_create_super_user.py

```diff
@@ -1,16 +1,24 @@
 from __future__ import unicode_literals
 
 from django.db import migrations, models
-from django.contrib.auth.models import User
+from django.contrib.auth.hashers import make_password
 import os
 
 def migrate_up(apps, schema_editor):
```

Comment:
These are not terrible names for data migration methods, but not terribly great either. Django doesn't do any Railsy magic to find the functions to run for forward/backward migrations: you have to explicitly pass them to the `RunPython` instance. That means you can give these more descriptive names, like `create_default_superuser` and `remove_default_superuser` or something.